cdap-ui/server/config/development/cdap.json

```diff
@@ -7,7 +7,7 @@
   "dashboard.ssl.disable.cert.check": "false",
   "ssl.external.enabled": "false",
   "router.server.address": "127.0.0.1",
-  "dashboard.bind.port": "11011",
```

Comment:
Do we want to commit this? (I'm fine with it, BTW - it can be useful to have the built version and the local changes running at the same time.)

Reply:
No this is a placeholder change to have the PR for base branch. All other PRs related to pipeline comments gets merged to this branch and finally we merge this PR to develop